src/main/resources/config/liquibase/migrationSQL/newPlagiarismCases.sql

```diff
@@ -0,0 +1,99 @@
+-- CREATE TEMPORARY TABLE WITH NEW PLAGIARISM CASES TO CREATE
+CREATE TEMPORARY TABLE new_plagiarism_case
+SELECT
+    exercise_id,
+    student_id,
+    GROUP_CONCAT(instructor_statement SEPARATOR ' ') as post,
+    GROUP_CONCAT(student_statement SEPARATOR ' ') as answer,
+    CASE
+        WHEN MAX(status) > 0 THEN 'PLAGIARISM'
+        WHEN MAX(status) = 0 THEN 'WARNING'
+        END as verdict,
```

Comment:
I think this should be one indentation further to the left. But since this will most likely change the checksum, which will mess with the test servers you can ignore this.
```suggestion
    END as verdict,
```